cli.js

```diff
@@ -158,7 +158,12 @@ function exit(results) {
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
 
-	process.exit(failed > 0 ? 1 : 0);
+	// Individually kill each child process.
+	Promise.map(results, function (result) {
+		return result.kill();
```

Comment:
Why are you using promises here? `cp.kill()` is a synchronous operation. Just do a `[].forEach()`.